Content.Server/_DV/Abilities/Chitinid/CoughingUpChitziteComponent.cs

```diff
@@ -0,0 +1,11 @@
+namespace Content.Server.Abilities.Chitinid;
+
+[RegisterComponent]
+public sealed partial class CoughingUpChitziteComponent : Component
+{
+    [DataField("accumulator")]
+    public float Accumulator = 0f;
+
+    [DataField("coughUpTime")]
```

Comment:
just [DataField] and replace accumulator with an AutoPausedField TimeSpan NextCough using TimeOffsetSerializer